message/tests.py

```diff
@@ -0,0 +1,114 @@
+from django.contrib.auth import get_user_model
+from player.models import Player, BallGame
+from game_event.models import GameEvent
+from court.models import Court
+from message.models import Message
+from decimal import Decimal
+from django.core.exceptions import ValidationError
+from datetime import datetime
+import pytest
+
+TEST_ID = 2
+TEST_TIME = datetime.today()
```

Comment:
I noticed @MaayanMashhadi just created a PR for moving away from using this library. Can you check if this is relevant here as well?

Reply:
yes, we get warnings as well. @yftacherzog

Reply:
Do you want to address it before this is merged?

Reply:
@MaayanMashhadi's PR will make us change some of the tests, because we are using datetime library there.

Reply:
> Do you want to address it before this is merged?

yes. handling the issue right now.

Reply:
@yftacherzog oh ok, got it. 
I changed the same issue at our `message/tests.py` .